server/service/src/sync/synchroniser.rs

```diff
@@ -295,7 +279,7 @@ impl Synchroniser {
                 false => Some(logger),
                 true => None,
             },
-            central_sync_server_id, // Also include OMS Central as source site_id is null, should probably be passing a vec here...
+            None,
```

Comment:
With the change to `sync_buffer.rs` above, won't this just get back the null source_site_id records?
We could make passing null not add a source_site_id filter at all?